src/psycop_model_evaluation/binary/time/timedelta_plots.py

```diff
@@ -167,6 +168,7 @@ def plot_time_from_first_positive_to_event(
         eval_dataset: EvalDataset object
         min_n_in_bin (int): Minimum number of patients in each bin. If fewer, bin is dropped.
         bins (Sequence[float]): Bins to group by. Defaults to (5, 25, 35, 50, 70).
+        bin_unit (Literal["H", "D", "M", "Q", "Y"], optional): Unit of time to bin by. Defaults to "M".
```

Comment:
```suggestion
        bin_unit (Literal["h", "D", "M", "Q", "Y"]): Unit of time to bin by. Defaults to "M".
```

Not optional, e.g. doesn't accept `None`.